python/pyspark/ml/classification.py

```diff
@@ -1275,7 +1263,6 @@ def __init__(
     ):
         ...
 
-    @keyword_only
```

Comment:
I don't think so ? It saves params to `self._input_kwargs` and we use `self._input_kwargs` in following code

Reply:
We should manually store them, or use a differnet decorator then. Let me fix it up.